source/reference/collection-level-filtering/filter-regex.txt

```diff
@@ -36,6 +36,61 @@ you can use regular expressions:
       }
    }
 
+The regular expression pattern you pass into a filter must follow the :query:`regex <$regex>` syntax supported by the MongoDB server.
```

Comment:
Can we just link to the server’s own regexp docs? That seems better than duplicating the information here.